oshdb-tool/etl/src/main/java/org/heigit/bigspatialdata/oshdb/tool/importer/extract/TypeStartFinder.java

```diff
@@ -196,7 +196,7 @@ public static OSMType getType(PbfBlob blob) throws InvalidProtocolBufferExceptio
 			return OSMType.WAY;
 		if (group.getRelationsCount() > 0)
 			return OSMType.RELATION;
-		return OSMType.UNKNOWN;
+		return null;
```

Comment:
why do you not throw an exception like in the `fromInt` method below? Can't a `null` cause hard to understand `NullPointerException`s down the line?

Reply:
here would be a place where a switch statement dereferences a possibly `null` OSMType value: https://github.com/GIScience/oshdb/blob/remove-unknown-type/oshdb-tool/etl/src/main/java/org/heigit/bigspatialdata/oshdb/tool/importer/extract/TypeStartFinder.java#L52

Reply:
should be fixed now, it throws an exception instead of returning null, if getType could not find out the current OSMType.